lib/http2.rb

```diff
@@ -0,0 +1,13 @@
+require 'http2/version'
+require 'http2/error'
+require 'http2/emitter'
+require 'http2/buffer'
+require 'http2/flow_buffer'
+require 'http2/huffman'
+require 'http2/huffman_statemachine'
+require 'http2/compressor'
+require 'http2/framer'
+require 'http2/connection'
+require 'http2/client'
```

Comment:
This change will also break anyone that was using `require 'http/2/client`.  My guess is there isn't much usage like that since the README examples all have the top level file being required and it requires all other files.

Reply:
I don't see this in the SDK so I think we are fine. But you're right that someone could be doing this. Documented usage is different as you point out. I think it's fine personally.

Reply:
Seeing as this is not expected behavior, this one doesn't worry me, but I'll leave it to the author.